source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -4347,6 +4347,10 @@ export class DateInput extends Input {
         };
 
         if (this.defaultValue) {
+
+            if (this.defaultValue === "today") {
```

Comment:
If there is no customer, I prefer to cancel this PR. It introduces inconsistency like in ElementProperties. 

![image](https://user-images.githubusercontent.com/6290692/164304787-5b106f70-eb7d-47b9-a7ea-cb7c2a0369e4.png)

It breaks other features, for example, the today is changed to `4/20` after I drag two new items into the designer.

![image](https://user-images.githubusercontent.com/6290692/164305836-53ee5edf-0934-4070-893d-525f5f9e627e.png)

Reply:
This feature is broken too.

![image](https://user-images.githubusercontent.com/6290692/164306530-1418485d-230a-4e20-9afa-30e926c05855.png)

expected
![image](https://user-images.githubusercontent.com/6290692/164306734-58369700-0a53-40f5-a2e1-613126b8d693.png)

Reply:
I am seeing this feature work for me:

<img width="666" alt="image" src="https://user-images.githubusercontent.com/98650930/164312611-c3fbdc39-2db8-4c93-89d6-f6cd9ae935ed.png">

Reply:
@licanhua Changing the code to:

```
        if (this.defaultValue) {
            if (this.defaultValue === "today") {
                this._dateInputElement.value = this.currentDate();
            } else {
                this._dateInputElement.value = this.defaultValue;
            }
        }
```

appears to solve the first issue you pointed out